testsuite-osgi/pom.xml

```diff
@@ -240,6 +258,7 @@
             <configuration>
               <name>%bundle.symbolicName%.link</name>
               <value>%url%</value>
+              <separator>${line.separator}</separator>
```

Comment:
why this change ?

Reply:
without this change, the plugin fails because it needs to aggregate several dependencies and fails because the line separarator is not configured

https://github.com/veithen/alta/blob/5e528a17fae44285d3e446b9a9a12e1dc69525b3/src/main/java/com/github/veithen/alta/AbstractGenerateMojo.java#L276